MAINTENANCE.md

```diff
@@ -207,24 +198,23 @@ Guidelines on how to write release notes:
 
 - Go over all the PRs / commits since previous release. Usually: `git log --first-parent` to see
   merge commits and commits that are directly on a branch. You can also try filtering
-  merged PRs on Github by merge date, but it's cumbersome.
+  merged PRs on Github by merge date, but it's cumbersome. Since 1.0 we try to assign each merged PR to a milestone,
+  which should make this process much easier - you can just go over e.g. https://github.com/scylladb/scylla-rust-driver/milestone/18?closed=1
 
 - Items in release notes will usually correspond 1:1 to PRs / comits - but not always. It is possible that
   some functionality that should be a single item on the list is split over multiple PRs.
   It is also possible that single PR will be mentioned in two items.
 
-- Release notes for a a major version should contain all changes since the previous major version. Release notes for a minor version should contain all changes since previous version.
+- Release notes should contain all items since previous (in terms of semver) version released at the time. Items that were backported will be
+  duplicated between release notes for at least 2 different versions that way. It is fine, the backport should just be marked as such in release notes.
```

Comment:
:pick:
```suggestion
- Release notes should contain all items since previous (in terms of semver) version released at the time. Items that were backported will be
  duplicated between release notes for at least 2 different versions that way. This is fine, the backport should just be marked as such in release notes.
```

Reply:
Can you provide an illustrative real-life example?

Reply:
Example of what? Change that was backported, or marking it as such in release notes?

I intentionally did not specify how to mark it exactly. Since we did not do that before, I thought we can decide on that when the need arises.

Reply:
> Example of what? Change that was backported, or marking it as such in release notes?

I had in mind mainly the release notes part. A good imaginary backported change would be good to better illustrate the case, though.

> I intentionally did not specify how to mark it exactly. Since we did not do that before, I thought we can decide on that when the need arises.

Makes sense. Let's leave it as-is then.